backend/open_webui/retrieval/web/brave.py

```
@@ -12,31 +13,139 @@
+    }
+
+    # Prepare the request body for chat completions with streaming and citations enabled
+    payload = {
+        "stream": True,
+        "enable_citations": True,
+        "messages": [{"role": "user", "content": query}],
     }
-    params = {"q": query, "count": count}
 
-    response = requests.get(url, headers=headers, params=params)
+    response = requests.post(url, headers=headers, json=payload, stream=True)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)